modules/Sources/Features/BalanceBreakdown/BalanceBreakdownView.swift

```diff
@@ -221,9 +228,9 @@ extension BalanceBreakdownView {
                 shadowOffset: 6
             )
             .padding(.bottom, 15)
-            .disabled(!viewStore.isShieldableBalanceAvailable || viewStore.isShieldingFunds)
+            //.disabled(!viewStore.isShieldableBalanceAvailable || viewStore.isShieldingFunds)
```

Comment:
Is this temporary? If it is then add a TODO and file an issue, otherwise delete the commented-out code.

Reply:
It was temporary, disable reactivated for the final PR